### ISSUE TITLE
borschik:include не подключает JS-файлы

### ISSUE THREAD

**Issue by Genesis15** [bem/bem-forum-content-ru#679]
Добрый день.

Начал делать проект на bem + angular.
Для блока angular.js

/*borschik:include:../../libs/angular/angular.js*/
/*borschik:include:../../libs/angular-ui-router/release/angular-ui-router.js*/
// Hello World!

В index.js скрипт подключает, но файлы не инклюдит.

/* end: ../../libs/bem-ng/common.blocks/angular/angular.js */
/* begin: ../../service.blocks/angular/angular.js */
/*borschik:include:../../libs/angular/angular.js*/
/*borschik:include:../../libs/angular-ui-router/release/angular-ui-router.js*/
// Hello World!

Все библиотеки существую и лежат в lib

Подскажите, что не так? Спасибо.

**Comment by apsavin** [comment]
А для сборки используется `enb-borschik/techs/js-borschik-include` ?

**Comment by Genesis15** [comment]
Нет, как его туда добавить?
```js
var techs = {
//...
borschikJsIncludeTech: require('enb-borschik/techs/js-borschik-include'),
...
}
//...
// borschik
[techs.borschikJsIncludeTech, { target: '?.pre.js' }],
[techs.borschik, { source: '?.js', target: '?.min.js', minify: isProd }],
[techs.borschik, { source: '?.css', target: '?.min.css', tech: 'cleancss', minify: isProd }]```

**Comment by apsavin** [comment]
Да, как-то так. Может быть, еще sourceSuffixes передать вместе с target.

**Comment by Genesis15** [comment]
Спасибо
